modules/installation-azure-marketplace-subscribe.adoc

```diff
@@ -129,7 +129,39 @@ ifdef::ipi[]
 . Record the image details of your offer. You must update the `compute` section in the `install-config.yaml` file with values for `publisher`, `offer`, `sku`, and `version` before deploying the cluster.
 endif::ipi[]
 ifdef::upi[]
-. Record the image details of your offer. If you use the Azure Resource Manager (ARM) template to deploy your worker nodes, you can update `storageProfile.imageReference` by deleting the `id` parameter and adding the `offer`, `publisher`, `sku`, and `version` parameters using the values from your offer.
+. Record the image details of your offer. If you use the Azure Resource Manager (ARM) template to deploy your worker nodes:
++
+.. Update `storageProfile.imageReference` by deleting the `id` parameter and adding the `offer`, `publisher`, `sku`, and `version` parameters by using the values from your offer.
+.. Specify a `plan` for the virtual machines (VMs). 
++
+.Sample `06_workers.json` ARM template with an updated `storageProfile.imageReference` object and a specified `plan` 
++
+[source,json]
+----
+...
+  "plan" : {
+    "name": "rh-ocp-worker",
+    "product": "rh-ocp-worker",
+    "publisher": "redhat"
+  },
+  "dependsOn" : [
+    "[concat('Microsoft.Network/networkInterfaces/', concat(variables('vmNames')[copyIndex()], '-nic'))]"
+  ],
```

Comment:
On the code block in the preview, some of the characters (namely, "..." and "," are showing up in red. Is this intentional? (I'm still learning, so maybe it is!?)

Reply:
Well spotted, @eohartman . I removed the default substitutions, so these highlighted red entities no longer exist.